bot/cogs/geo_cog/geo_main.py

```diff
@@ -0,0 +1,193 @@
+        else:
+            random_sample[0] = execute[3]
+
+        # Grab location and URL data
+        args['location'] = f"{execute[1]},{execute[2]}"
+
+        # Some entries have a city name attached, we retrieve that here.
+        full_name = ""
+        if len(str(self.get_parameter(random_sample[0], "city"))) > 1 and not "nan":
+            full_name = str(self.get_parameter(random_sample[0], "city"))
+            full_name += " "
+            full_name += str(self.get_parameter(random_sample[0], "country"))
+
+        labels_and_emojis: dict[str] = {
+            'labels': [],
+            'emojis': []
+        }
+
+        for i in range(5):
+            iso3 = random_sample[i]
+            labels_and_emojis['labels'].append(self.get_parameter(iso3, "country"))
+            labels_and_emojis['emojis'].append(self.get_country_emoji(self.get_parameter(iso3, "iso2")))
+
+        new_geo_view = GeoView(args, labels_and_emojis, full_name)
+
+        # Create the initial embed.
+        initial_embed = await new_geo_view.create_embed(api_res_time, " ")
+        file = discord.File(fp=f'bot/cogs/geo_cog/temp_assets/{file_name}', filename=file_name)
+        await ctx.send(files=[file, initial_embed[1]], embed=initial_embed[0], view=new_geo_view)
+        await new_geo_view.wait()
+
+    @ext.command()
+    @commands.cooldown(1, 30, commands.BucketType.user)
+    @ext.example("?geoguess lb")
+    @ext.long_help("Displays the GeoguessrLeaderboard")
+    @ext.short_help("GeoguessrLeaderboard")
+    async def lb(self, ctx) -> None:
+        guild = self.bot.get_guild(ctx.guild.id)
+        new_embed = discord.Embed(color=0x00FF61)
+        new_embed.set_author(name="Discord Geoguessr by @yeet.us",
+                             icon_url="https://i.imgur.com/ZyGOyTg.png")
+        lb_output = ""
+
+        entries: int = 10 if (await self.repo.return_size()) >= 10 else (await self.repo.return_size())
+        rank_btn = Button(label="What's my rank?", style=discord.ButtonStyle.primary, row=1)
+
+        async def view_rank(interaction: discord.Interaction) -> None:
+            if await self.repo.get_by_userid_scores_descending(interaction.user.id) is not None:
+                size = (await self.repo.return_size())
+                rank = 0
+
+                for x in range(size):
+                    if (await self.repo.get_rank())[x].get('user_id') == interaction.user.id:
+                        rank = (await self.repo.get_rank())[x].get('RANK')
+
+                score = (await self.repo.sort_and_return())[rank - 1].get('score')
+                msg = f'You are ranked **#{rank}** and you have **{score} points**'
+                await interaction.response.send_message(msg, ephemeral=True)
+            else:
+                await interaction.response.send_message(f'You aren\'t even on the board, '
+                                                        f'try a game out! Run ?geoguess game', ephemeral=True)
```

Comment:
Remove the `f` portion and change it to `"<string>"` so you don't have to escape the apostrophes.